x/poe/bootstrap_integration_test.go

```diff
@@ -0,0 +1,129 @@
+package poe_test
+
+import (
+	"encoding/json"
+	"sort"
+	"testing"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	fuzz "github.com/google/gofuzz"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/confio/tgrade/x/poe/keeper"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/confio/tgrade/x/poe"
+	"github.com/confio/tgrade/x/poe/contract"
+	"github.com/confio/tgrade/x/poe/types"
+)
+
+func TestIntegrationBootstrapPoEContracts(t *testing.T) {
+	// setup contracts and seed some data
+	ctx, example, _, members := setupPoEContracts(t)
+
+	ocContractAddr, err := example.PoEKeeper.GetPoEContractAddress(ctx, types.PoEContractTypeOversightCommunity)
+	require.NoError(t, err)
+	tcAdapter := contract.NewTrustedCircleContractAdapter(ocContractAddr, example.TWasmKeeper, nil)
+
+	voters, err := tcAdapter.QueryListVoters(ctx)
+	require.NoError(t, err)
+
+	//verify the members list are all voting members
+	notVotingMembers := make(map[string]struct{}, len(members))
+	for _, m := range members {
+		notVotingMembers[m] = struct{}{}
+	}
+
+	for _, v := range voters.Members {
+		delete(notVotingMembers, v.Addr)
+	}
+	assert.Empty(t, notVotingMembers)
```

Comment:
I got confused with the "notVotingMembers" here. But this test ensures that all OC members are voting members.